src/main/ArbitrageExecution.js

```diff
@@ -91,6 +91,10 @@ const ArbitrageExecution = {
             logger.execution.trace(`Blocking execution because ${ArbitrageExecution.executedTradesInLastXSeconds(10)} trades have been completed in the last 10 seconds`);
             return false;
         }
+        if (ArbitrageExecution.executedTradesInLastXSeconds(1) >= 9) {
```

Comment:
You're tracking towards the right idea. This won't protect a user because a successful position needs three trades. In your example you check if there is at least one trade available that won't exceed the 10 trade limit, but you need to ensure 3 are available or the position will fail after the first trade.

Reply:
oh.... I got it. 
Thank you for the reply ;)